conf/base.config

```diff
@@ -0,0 +1,59 @@
+process {
+  executor = 'local'
+  cpus = { 2 }
+  memory = { 8.GB }
+  time = { 2.h }
+
+  errorStrategy = { task.exitStatus == 143 ? 'retry' : 'finish' }
```

Comment:
Do need `{}` on this one as it has a variable and needs to be evaluated later, so leave this as it is.